setup.py

```diff
@@ -221,10 +222,11 @@ def params():
     packages = ["netconnectd"]
     zip_safe = False
 
-    dependency_links = [
-        "https://github.com/ManuelMcLure/wifi/tarball/master#egg=wifi-1.1.0"
```

Comment:
should this stay on dependency links? Or did something complain about it?
https://python-packaging.readthedocs.io/en/latest/dependencies.html#packages-not-on-pypi